Fruitties/shared/src/iosMain/kotlin/com/example/fruitties/di/viewmodel/ViewModelStoreUtil.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.fruitties.di.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelStore
+import androidx.lifecycle.viewmodel.CreationExtras
+import kotlinx.cinterop.BetaInteropApi
+import kotlinx.cinterop.ObjCClass
+import kotlinx.cinterop.getOriginalKotlinClass
+import kotlin.reflect.KClass
+
+@Suppress("unused") // Android Studio is not aware of iOS usage.
```

Comment:
I'm not sure I understand the need for this method. Could you expand the code comment to explain why is this required?